Objects/Objects/Structural/CSI/Geometry/CSIElement1D.cs

```diff
@@ -112,7 +144,14 @@ public CSIElement1D() { }
 
   public string PierAssignment { get; set; }
   public string SpandrelAssignment { get; set; }
-  public double[]? Modifiers { get; set; }
+
+  [JsonIgnore, Obsolete("This is changed to a list of doubles for Grasshopper compatibility")]
+  public double[]? Modifiers
```

Comment:
Just FYI, you could have changed `Modifiers` to be a `List<double>?` without introducing a new property, since our serializer handles `double[]` and `List<double>` exactly the same (i.e. it wouldn't have broken the JSON layer)

But.. there's nothing wrong with doing it the way you have it here either, so we're fine.

Reply:
Good to know, thanks @JR-Morgan!